client/dashboard/app/routes/applications.technical-project.tsx

```diff
@@ -35,9 +36,16 @@ export const meta: V2_MetaFunction = () => {
 
 export const links = () => [...projectTilesLinks(), ...projectLinks()];
 
-export const loader: LoaderFunction = ({ request }) => {
+export const loader: LoaderFunction = async ({ request }) => {
   const url = new URL(request.url);
   const slug = url.searchParams.get("slug");
+  const userDetails = await getUserDetails(request);
```

Comment:
does this check for technicalp specifically? or any domain. Can you check what happens when you have filled the form for events and try to fill techp